pkg/test/e2e/cilium/testdata/connectivity-check.yaml

```diff
@@ -0,0 +1,1039 @@
+# Automatically generated by Makefile. DO NOT EDIT
```

Comment:
If we are copying this file, we may use the one with my node-local-dns fix: https://github.com/rastislavs/cilium/blob/connectivity-check-dns/examples/kubernetes/connectivity-check/connectivity-check.yaml
With that one all tests should pass. We just need to document its source somewhere :)